crates/utils/src/error.rs

```diff
@@ -162,6 +159,38 @@ pub enum LemmyErrorType {
   },
   CouldntParsePaginationToken,
   PluginError(String),
+  InvalidFetchLimit,
+  CouldntCreateCommentReply,
+  CouldntUpdateCommentReply,
+  CouldntMarkCommentReplyAsRead,
+  CouldntCreateEmoji,
+  CouldntUpdateEmoji,
+  CouldntCreatePerson,
+  CouldntUpdatePerson,
+  CouldntCreateModlog,
+  CouldntUpdateModlog,
+  CouldntCreateSite,
+  CouldntUpdateSite,
+  CouldntCreateRegistrationApplication,
+  CouldntUpdateRegistrationApplication,
+  CouldntCreateTag,
+  CouldntUpdateTag,
+  CouldntCreatePostTag,
+  CouldntUpdatePostTag,
+  CouldntCreateTagline,
+  CouldntUpdateTagline,
+  CouldntCreateImage,
+  CouldntAllowInstance,
+  CouldntBlockInstance,
+  CouldntInsertActivity,
+  CouldntCreateRateLimit,
+  CouldntCreateCaptchaAnswer,
+  CouldntUpdateFederationQueueState,
+  CouldntCreateOauthAccount,
+  CouldntCreatePasswordResetRequest,
+  CouldntCreateLoginToken,
+  CouldntUpdateLocalSiteUrlBlocklist,
+  CouldntCreateEmailVerification,
```

Comment:
Is it really necessary to have separate error messages for all of these?

Reply:
I think its worth it, mainly because a failed create denotes a constraint conflict, while a failed update means its missing, or something else. They're already done anyway.